builder-base/Dockerfile

```diff
@@ -12,13 +12,54 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG BASE_IMAGE
+ARG BASE_IMAGE=amazonlinux:2
+
 FROM ${BASE_IMAGE}
 
-COPY golang-checksum /golang-checksum
-COPY buildkit-checksum /buildkit-checksum
-COPY install.sh /opt/install.sh
```

Comment:
We need the install.sh script as well because we use it to bootstrap our container to build our builder-base image in the prow job here: https://github.com/vivek-koppuru/eks-distro-prow-jobs/blob/main/jobs/aws/eks-distro-build-tooling/builder-base-postsubmits.yaml#L40

Reply:
+1